### PR TITLE
feat(suite): eth staking custom gas price

### LINKED ISSUE #11564: Custom gas price for Ethereum Staking, Unstaking and Claming
**Design:**
Screenshot 2024-03-13 at 14 13 36
Screenshot 2024-03-13 at 14 18 13

### PR DESCRIPTION
**Pull request by martykan**
## Description

Add support for custom gas price in staking forms. 
Implementation is based on Invity forms, uses existing Fees component instead of FeesInfo.

## Related Issue

Resolve #11564

## Screenshots:

Screenshot 2024-03-26 at 12 37 12

### CODE REVIEW

**Comment by martykan** [comment]
I also included a change in Fees component layout, specifically I moved the FeeDetails component below the options bar and made it hidden in case of custom fees. 
The goal was to reduce layout shift when FeeDetails appear/disappear and duplicate information between those and the form.
This also affects other forms that use the Fees component. 

## Before
Screenshot 2024-03-26 at 12 37 12 2

## After
Screenshot 2024-03-26 at 14 00 34
Screenshot 2024-03-26 at 14 00 41

File: packages/suite/src/components/wallet/Fees/Fees.tsx [hunk 1]
```diff
@@ -162,6 +182,27 @@ export const Fees = <TFieldValues extends FormState>({
...
+                            height: { duration: 0.2, ease: motionEasing.transition },
+                            marginTop: { duration: 0.25, ease: motionEasing.transition },
+                        }}
+                    >
+                        <FeeDetails
+                            networkType={networkType}
+                            feeInfo={feeInfo}
+                            selectedLevel={selectedLevel}
+                            transactionInfo={transactionInfo}
+                        />
```
Comment (tomasklim) [inline packages/suite/src/components/wallet/Fees/Fees.tsx:204]: This change is great 🥳

**Review by tomasklim — APPROVED** [review]

**Review by sime — APPROVED** [review]
Happy path works.

**Comment by marekrjpolak** [comment]
Should this be visible in BTC account? 
image

**Review by marekrjpolak — APPROVED** [review]
LGTM

**Pull request merged**